docs/experiment/sdks/ruby-sdk.md

```diff
@@ -168,6 +171,156 @@ experiment.fetch_async(user) do |_, variants|
 end
 ```
 
+## Local evaluation
+
+Implements evaluating variants for a user via [local evaluation](../general/evaluation/local-evaluation.md). If you plan on using local evaluation, you should [understand the tradeoffs](../general/evaluation/local-evaluation.md#targeting-capabilities).
+
+!!!warning "Local Evaluation Mode"
+    The local evaluation client can only evaluation flags which are [set to local evaluation mode](../guides/create-local-evaluation-flag.md).
+
+### Install
+
+Install the Ruby Server SDK's local evaluation.
+
+!!!warning "OS, and architecture support"
+    Local evaluation requires `CGO` be enabled (`CGO_ENABLED=1`). Additionally, the local evaluation package currently only supports the following OS' and architectures (`GOOS/GOARCH`):
```

Comment:
cgo?

Reply:
I saw python also includes the `cgo` description. Should that also be removed?

Reply:
Good catch, I removed from the title and forgot in the text, I will create a PR to clean this up